db/migrate/20190705195039_add_constraints_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddConstraintsToUser < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null(:users, :name, false)
```

Comment:
Well, here are the constraints 👏. Please run annotate again since the annotations for user model looks like out of date.